tests/test_PropertyList.py

```diff
@@ -321,34 +319,28 @@ def testHierarchy(self):
         apl.set("CURRENT", 49.5)
         apl.set("CURRENT.foo", -32)
         apl.set("CURRENT.bar", 2)
-        with self.assertWarns(DeprecationWarning):
-            self.assertEqual(apl.get("CURRENT"), 49.5)
+        self.assertEqual(apl.get("CURRENT"), 49.5)
```

Comment:
Everywhere you added a test for `get` I suggest you also add a test for [key]. It seems a natural fit.

Reply:
I had added explicit tests for defaulting get but I just realized I was missing a test with a default default so I've added one for that to PropertySet and PropertyList.  I don't think I gain anything by adding a get with a missing keyword after every get test.